### ISSUE TITLE
Implement metricq-summary in Python

### ISSUE THREAD

**Issue by tilsche** [metricq/metricq-python#13]
Having a python variant of `metricq-summary` would be great for environments where it is hard to compile software. Yes that is weird. Looking at you taurus-ml.

But it could be a good example for a drain. Anyway... maybe this will be wontfix. We'll see.

**Comment by tilsche** [comment]
This looks like a good start for @Daddelhai. Please make a branch here about this. The desired functionality can be found in https://github.com/metricq/metricq-cpp/tree/master/tools/metricq-summary/src - although cool ideas of additional information to show or options to include are also welcome. Take a look at our existing tools in https://github.com/metricq/metricq-python/tree/master/tools to see how we handle cmdline arguments.

**Comment by tilsche** [comment]
We may want to use a different name here to avoid conflicts with the C++ version of the program. I assume at some point the tools should also be installed by `setup.py`.

**Comment by tilsche** [comment]
Oh well, this depends on #12. That may be a bit more complicated to get started.
